Lib9c/Module/ActionPointModule.cs

```diff
@@ -0,0 +1,27 @@
+using Bencodex.Types;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+
+namespace Nekoyume.Module
+{
+    public static class ActionPointModule
+    {
+        public static long GetActionPoint(this IWorldState worldState, Address address)
+        {
+            var value = worldState.GetAccountState(Addresses.ActionPoint).GetState(address);
+            if (value is Integer integer)
+            {
+                return integer;
+            }
+
+            return 0;
+        }
+
+        public static IWorld SetActionPoint(this IWorld world, Address address, long actionPoint)
+        {
+            var account = world.GetAccount(Addresses.ActionPoint);
+            account = account.SetState(address, (Integer)actionPoint);
```

Comment:
```suggestion
        public static long GetActionPoint(this IWorldState worldState, Address avatarAddress)
        {
            var value = worldState.GetAccountState(Addresses.ActionPoint).GetState(avatarAddress);
            if (value is Integer integer)
            {
                return integer;
            }

            return 0;
        }

        public static IWorld SetActionPoint(this IWorld world, Address avatarAddress, long actionPoint)
        {
            var account = world.GetAccount(Addresses.ActionPoint);
            account = account.SetState(avatarAddress, (Integer)actionPoint);
```
명시적으로 적어두면 편할 것 같습니다.